### PR TITLE
Don't use deprecated ATOMIC_USIZE_INIT

### PR DESCRIPTION
**Pull request by EPashkin**
cc @sdroege

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -226,9 +226,9 @@ pub use source_futures::*;
 // This works around it by using our own counter for threads.
 //
 // Taken from the fragile crate
-use std::sync::atomic::{AtomicUsize, Ordering, ATOMIC_USIZE_INIT};
+use std::sync::atomic::{AtomicUsize, Ordering};
 fn next_thread_id() -> usize {
-    static mut COUNTER: AtomicUsize = ATOMIC_USIZE_INIT;
+    static mut COUNTER: AtomicUsize = AtomicUsize::new(0);
```
Comment (sdroege) [inline src/lib.rs]: Ah is this stabilized since 1.31 already (`const`ified constructor)? I thought it was later :)

:+1: if it builds with 1.31

**Comment by GuillaumeGomez** [comment]
👍

**Comment by sdroege** [comment]
@GuillaumeGomez the examples repo is confused with regards to versions, can you update that? :) See CI failures

**Comment by EPashkin** [comment]
@sdroege IMHO problem in this: https://github.com/gtk-rs/examples/pull/233#issuecomment-466269884

**Comment by sdroege** [comment]
Indeed

**Comment by EPashkin** [comment]
CI passed

**Pull request merged**